### ISSUE TITLE
No rule to make target 

### ISSUE THREAD

**Issue by martinakos** [NotixApp/Journal#121]
I'm trying to build your project but I keep getting this error:

make[1]: *** No rule to make target '/home/martin/tool_tests/Journal/build/journal/../qtmypaint/libqtmypaint.a', needed by 'Journal'. Stop.

any ideas what could be the problem?

**Comment by lordadamson** [comment]
yes. The project qtmypaint is no longer there. I forgot to remove it from the .pro

Will fix soon, unless you wanna help out :smile:

**Comment by martinakos** [comment]
I managed to trick it by creating a folder ~/Journal/release/qtmypaint and copying liblibmypaint.a renamed as libqtmypaint.a
It builds and runs. I could draw and write text, but I couldn't paste screen captures in ubuntu. 
I'm looking for some onenote replacement for linux. I would be great if you could combine efforts with [Cherrytree](https://github.com/giuspen/cherrytree). I read in some issue that they intent to move to Qt5

**Comment by lordadamson** [comment]
by pasting do you mean, ctrl-c/ctrl-v?
You can drag the screen capture and drop it to journal and it should work.

I'll check them out :smile:

**Comment by martinakos** [comment]
yes, crtl-c/ctrl-v. Actually in ubuntu I do Crtl+Shitf+Prnt Scr/Ctrl+v . I can paste the screen capture successfully in this way in other programs

**Comment by lordadamson** [comment]
Oh okay, sounds like a nice feature. Should be a separate issue though.

On May 28, 2018 5:19 PM, martinakos <notifications@github.com> wrote:

yes, crtl-c/ctrl-v. Actually in ubuntu I do Crtl+Shitf+Prnt Scr/Ctrl+v . I can paste the screen capture successfully in this way in other programs
